content/blog/creating-a-python-aws-application-using-flask-and-redis/index.md

```diff
@@ -15,7 +15,7 @@ I've recently started developing with Pulumi, and have begun to explore its inne
 
 This blog post recreates the existing [Typescript voting app example](https://www.pulumi.com/docs/tutorials/aws/aws-ts-voting-app/) step by step in Python with Flask as the frontend and Redis as the backend. In future blog posts, we will explore how to change the front and backends, how to upgrade the app with additional AWS services, and migrating from one cloud provider to another.
 
-The first few lines of our new `__main__.py` file indicate which libraries to import and describe a pair of configuration options used by the application
+The project structure of our new Python voting app will be similar to the Typescript example, and the biggest difference will be the code describing the cloud infrastructure. Instead of an `index.ts` file, Pulumi will be given a new `__main__.py` file. The first few lines of the file indicate which libraries to import and describe a pair of configuration options used by the application.
```

Comment:
```suggestion
The project structure of our new Python voting app is similar to the Typescript example and the biggest difference is the code describing the cloud infrastructure. Instead of an `index.ts` file, the project uses `__main__.py` file. The first few lines of the file indicate which libraries to import and describe a pair of configuration options used by the application.
```

Reply:
I think we should just talk about `pulumi new` here. The `__main__.py` isn't the only difference. The Pulumi.yaml has a `runtime: python` and we also need a requirements.txt file for the dependencies

If we scaffold a `pulumi new aws-python` app, we have all the components we need and it will help the users be able to recreate easily

$0.02

Reply:
Added a small tutorial on how to create the project, and incorporated Sophia's edits.